Overview.src.html

```diff
@@ -336,17 +336,35 @@ <h4 id=terminology-headers>Headers</h4>
 <hr>
 
 <p>A <dfn>simple header</dfn> is a <span title=concept-header>header</span> whose
-<span title=concept-header-name>name</span> is `<code title>Accept</code>`,
-`<code title>Accept-Language</code>`, or `<code title>Content-Language</code>`, or whose
-<span title=concept-header-name>name</span> is `<code title>Content-Type</code>` and
-<span title=concept-header-value>value</span>, <span title=concept-header-parse>once parsed</span>,
-has a MIME type (ignoring parameters) that is
-`<code title>application/x-www-form-urlencoded</code>`, `<code title>multipart/form-data</code>`, or
-`<code title>text/plain</code>`.
+<span title=concept-header-name>name</span> is one of:
```

Comment:
I think it's fine to leave off the ":" here since it's basically one long paragraph, with nested lists. (HTML cannot express that unfortunately due to parsing issues, but this will have to do.)